src/main/java/com/hydratereminder/command/reset/ResetCommandHandler.java

```diff
@@ -1,15 +1,35 @@
 package com.hydratereminder.command.reset;
 
+import com.hydratereminder.HydrateReminderPlugin;
+import com.hydratereminder.chat.ChatMessageSender;
+import java.time.Instant;
+import java.util.Optional;
+import javax.inject.Inject;
 import javax.inject.Singleton;
 
 import com.hydratereminder.command.CommandHandler;
 
 @Singleton
 public class ResetCommandHandler implements CommandHandler {
 
+    @Inject
+    private HydrateReminderPlugin hydrateReminderPlugin;
```

Comment:
You can use a constructor based injection, and make this fields `private final`

Reply:
Updated